.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
In the future if you use `git add main.py` instead of `git add .` you can leave files like this out of your commits.